playbooks/precheck.yml

```diff
@@ -71,6 +72,26 @@
       when:
         - check_existing_k8s_service
 
+    - name: Perform fetch_os_kernel_version
+      shell: bash -c "uname -r"
```

Comment:
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/network_plugin/cilium/tasks/check.yml#L23-L30

You can use the ansible_kernel variable to get the kernel version directly.